port_ocean/tests/test_metric.py

```diff
@@ -0,0 +1,59 @@
+import json
+import pytest
+
+
+@pytest.mark.metric
+def test_metrics() -> None:
+    """
+    Test that the metrics logged in /tmp/ocean/metric.log match expected values.
+    """
+
+    log_path = "/tmp/ocean/metric.log"
+    delay = 2
+    batch_size = 400
+    total_objects = 2000
+    magic_string = "integration metrics"
+
+    with open(log_path, "r") as file:
+        content = file.read()
+
+    assert magic_string in content, f"'{magic_string}' not found in {log_path}"
+
+    start_idx = content.rfind(magic_string)
+    content_after_magic = content[start_idx + len(magic_string) :]
+
+    obj = json.loads(content_after_magic)
+    metrics = obj.get("metrics")
+    assert metrics, "No 'metrics' key found in the parsed JSON."
+
+    assert "fake-person" in metrics, "'fake-person' key missing in metrics data."
+    fake_person = metrics["fake-person"]
```

Comment:
what `fake-person` mean in a metric? I understand that its contains the `extract`, `load` `transform` but I wonder how we can monitor that?